redbot/cogs/warnings/warnings.py

```diff
@@ -76,6 +83,43 @@ def __init__(self, bot: Red):
         else:
             await ctx.send(_("Custom reasons have been disabled."))
 
+    @warningset.command()
+    @commands.guild_only()
+    async def toggledm(self, ctx: commands.Context):
+        """Toggle warns being sent to the users DM's"""
+        guild = ctx.guild
+        toggle = await self.config.guild(guild).toggle_dm()
+        await self.config.guild(guild).toggle_dm.set(not toggle)
+        toggle = not toggle
+        if toggle == True:
```

Comment:
You can change it to just:
```suggestion
        if toggle:
```
Or even not assign line above and just do `if not toggle:`